Source/Details/_ASDisplayView.mm

```diff
@@ -78,7 +78,10 @@ @implementation _ASDisplayView
   BOOL _inPointInside;
 
   NSArray *_accessibilityElements;
-  CGRect _lastAccessibilityElementsFrame;
+  BOOL _inAccessibilityElementCount;
```

Comment:
Let's use bitfield packing for all these. Could also combine them with the _inHitTest and _inPointInside options.

Reply:
Done

Reply:
Done. `_inHitTest` and `_inPointInside`  will happen within #1522